diff --git a/node_modules/@web3-onboard/core/dist/index-10054806.js b/node_modules/@web3-onboard/core/dist/index-10054806.js
index 38f2099..cb36875 100644
--- a/node_modules/@web3-onboard/core/dist/index-10054806.js
+++ b/node_modules/@web3-onboard/core/dist/index-10054806.js
@@ -2762,7 +2762,7 @@ async function getProvider(chain) {
         const { createPublicClient, http } = await import('viem');
         const publicProvider = createPublicClient({
             chain: viemChain,
-            transport: http()
+            transport: http(chain.rpcUrl)
         });
         viemProviders[chain.rpcUrl] = publicProvider;
     }
